<ui version="4.0" >
 <class>ProxyDialog</class>
 <widget class="QDialog" name="ProxyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>390</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Proxy settings</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Proxy type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox" >
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text" >
            <string>Socks5</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Http</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit" />
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox" >
          <property name="minimumSize" >
           <size>
            <width>30</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximum" >
           <number>65535</number>
          </property>
          <property name="value" >
           <number>8080</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>Proxy needs authentication</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2" />
        </item>
       </layout>
      </item>
      <item row="4" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>350</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
